librz/include/rz_core.h

```diff
@@ -397,6 +397,7 @@ typedef int (*RzCoreSearchCallback)(RzCore *core, ut64 from, ut8 *buf, int len);
 RZ_API int rz_core_bind(RzCore *core, RzCoreBind *bnd);
 
```

Comment:
Maybe a comment here that these are about ROP gadgets

Reply:
ROP Gadgets? I'm sorry, which one are you referring to? 
Both of them are related to the these print gadgets, right?

Reply:
Yes, ignore my comment. It has nothing to do with ROP - I checked the code and it's just a "print gadgets". Probably worth documenting what exactly it is - "printing a result of some command at the specified screen position" or something like that.